server/methods/accounts/accounts.app-test.js

```diff
@@ -5,24 +5,21 @@ import { Meteor } from "meteor/meteor";
 import { Factory } from "meteor/dburles:factory";
 import { check, Match } from "meteor/check";
 import { Random } from "meteor/random";
-import { Accounts as MeteorAccount } from "meteor/accounts-base";
+import { Accounts as MeteorAccounts } from "meteor/accounts-base";
 import { expect } from "meteor/practicalmeteor:chai";
 import { sinon } from "meteor/practicalmeteor:sinon";
-import { Accounts, Packages, Orders, Products, Shops, Cart } from "/lib/collections";
-import { Reaction } from "/server/api";
+import { SSR } from "meteor/meteorhacks:ssr";
+import { Accounts, Groups, Packages, Orders, Products, Shops, Cart } from "/lib/collections";
+import { Logger, Reaction } from "/server/api";
 import { getShop, getAddress } from "/server/imports/fixtures/shops";
 import Fixtures from "/server/imports/fixtures";
 
 Fixtures();
 
-before(function () {
-  this.timeout(10000);
-  Meteor._sleepForMs(7000);
-});
-
 describe("Account Meteor method ", function () {
   let shopId;
-  const fakeUser = Factory.create("account");
+  let fakeUser;
+  let fakeAccount;
```

Comment:
instead of creating a single user for all tests, create a new one for each test. this reduces the chances of one test impacting another